scripts/generate-changelog.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const replace = require('replace-in-file')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+const spawn = util.promisify(require('child_process').spawn)
+
+const name = require('emoji-name-map')
```

Comment:
```suggestion
const emojiNameMap = require('emoji-name-map')
```
in order to be clearer when using it